docs/documentation/functions/aria-suite/aria-operations-networks/New-AriaNetworksNsxtDataSource.md

```diff
@@ -0,0 +1,177 @@
+# New-AriaNetworksNsxtDataSource
+
+## Synopsis
+
+Add a new NSX-T Data Source to VMware Aria Operations for Networks.
+
+## Syntax
+
+### credentials (Default)
+
+```powershell
+New-AriaNetworksNsxtDataSource -fqdn <String> [-username <String>] [-password <String>] -nickname <String>
+ -collectorId <String> -enabled <String> [<CommonParameters>]
+```
+
+### certificate
+
+```powershell
+New-AriaNetworksNsxtDataSource -fqdn <String> [-certificate <String>] [-privatekey <String>] -nickname <String>
+ -collectorId <String> -enabled <String> [<CommonParameters>]
+```
+
+## Description
+
+The `New-AriaNetworksNsxtDataSource` cmdlet allows a user to add in a NSX-T Manager as a new data source in VMware Aria Operations for Networks in order to collect network flow data.
+
+## Examples
+
+### Example 1
+
+```powershell
+New-AriaNetworksNsxtDataSource -fqdn sfo-m01-nsx01.sfo.rainpole.io -certificate F:\certs\sfo-m01-nsx01.cer -privatekey F:\certs\sfo-m01-nsx01.key -nickname "sfo-m01-nsx01 - Management Domain NSX Manager" -CollectorId 15832:901:1711011916294613031 -enabled true
+```
+
+This example adds a NSX-T Manager server as a new data source in VMware Aria Operations for Networks by using a NSXT-T principal identity user.

```

Comment:
```suggestion
This example adds an NSX-T Manager server as a new data source in VMware Aria Operations for Networks by using a NSXT-T principal identity user.

```

Reply:
```suggestion
This example adds an NSX-T Manager server as a new data source in VMware Aria Operations for Networks by using a NSX principal identity user.

```

Reply:
Could we use "NSX" instead of NSX-T now?

Reply:
Function names (`...Nsxt...`) could remain.

Reply:
Resolved in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/954c74bc51b1fef11dc26b704cc60d52dd1bba80